rollbar-spring-boot/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    compile project(':rollbar-java')
+
+    implementation 'org.springframework:spring-web:5.2.6.RELEASE'
+    compile group: 'org.springframework', name: 'spring-webmvc', version: '4.1.6.RELEASE'
+
+    compile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.0.1'
```

Comment:
As far as versioning for dependencies you can centralize that at the top of the file. For example you can keep 
```
ext {
    spring-web-version = '4.0.0'
    spring-web-mvc-version = '1.0.0'
}
```
and then reference it like `"org.springframework:spring-webmvc:${spring-web-mvc-version}"`
That way you don't end up updating the dependency versions in different places.

Reply:
@mrunalk - i tried this.. it didn't compile for me.. do you have reference I can check out, an online example?

Reply:
Checkout tip #6 and #7 [here](https://medium.com/google-developer-experts/19-tips-for-gradle-in-android-projects-2019-edition-11af704eb06e),

Reply:
great, thanks! i updated the build file